cernopendata/modules/fixtures/data/records/cms-simulated-datasets-2012.json

```diff
@@ -202,7 +202,57 @@
-      "description": "<p>These data were processed in several steps:</p>\n<p><p><strong>Note</strong>\n        <br>\nTo get the exact LHE and generator's parameters, see <a href=\"/docs/cms-mc-production-overview\">CMS Monte Carlo production Overview</a>.</p>\n\n\n<strong>Step SIM</strong><br><a href=\"/record/11276\">Configuration file for SIM step config_0_1_cfg</a><br>Output dataset: /Diphoton2Jets_EW4_TuneZ2star_8TeV-madgraph-tauola/Summer12-START53_V7C-v2/GEN-SIM</p>\n\n<p><strong>Step HLT RECO</strong><br><a href=\"/record/11233\">Configuration file for HLT step HIG-Summer12DR53X-01394_1_cfg</a><br><a href=\"/record/11153\">Configuration file for RECO step HIG-Summer12DR53X-01394_2_cfg</a><br>Output dataset: /Diphoton2Jets_EW4_TuneZ2star_8TeV-madgraph-tauola/Summer12_DR53X-PU_RD1_START53_V7N-v1/AODSIM</p>\n"
+      "description": "<p>These data were generated in several steps:</p><p><strong>Note</strong><br>\n                              To get the exact LHE and generator's parameters, see <a href=\"/docs/cms-mc-production-overview\">CMS Monte Carlo production Overview</a></p>\n                           ",
+      "steps": [
+        {
+          "configuration_files": [
+            {
+              "script": "#!/bin/bash\nsource /cvmfs/cms.cern.ch/cmsset_default.sh\nexport SCRAM_ARCH=None\nif [ -r CMSSW_5_3_11_patch2/src ] ; then \n echo release CMSSW_5_3_11_patch2 already exists\nelse\nscram p CMSSW CMSSW_5_3_11_patch2\nfi\ncd CMSSW_5_3_11_patch2/src\neval `scram runtime -sh`\n\n\nscram b\ncd ../../\ncmsDriver.py step1 --filein \"dbs:/Diphoton2Jets_EW4_TuneZ2star_8TeV-madgraph-tauola/Summer12-START53_V7C-v2/GEN-SIM\" --fileout file:HIG-Summer12DR53X-01394_step1.root --pileup_input \"dbs:/MinBias_TuneZ2star_8TeV-pythia6/Summer12-START50_V13-v3/GEN-SIM\" --mc --eventcontent RAWSIM --runsScenarioForMC Run2012_AB_C_D_oneRunPerEra --pileup fromDB --datatier GEN-SIM-RAW --conditions START53_V7N::All --step DIGI,L1,DIGI2RAW,HLT:7E33v2 --python_filename HIG-Summer12DR53X-01394_1_cfg.py --no_exec --customise Configuration/DataProcessing/Utils.addMonitoring -n 288 || exit $? ; \n\ncmsDriver.py step2 --filein file:HIG-Summer12DR53X-01394_step1.root --fileout file:HIG-Summer12DR53X-01394.root --mc --eventcontent AODSIM,DQM --datatier AODSIM,DQM --conditions START53_V7N::All --step RAW2DIGI,L1Reco,RECO,VALIDATION:validation_prod,DQM:DQMOfflinePOGMC --python_filename HIG-Summer12DR53X-01394_2_cfg.py --no_exec --customise Configuration/DataProcessing/Utils.addMonitoring -n 288 || exit $? ; \n\n",
+              "type": "cmsDriver script"
+            },
+            {
+              "cms_confdb_id": "f83bc23f20bdf9090c24e193b7f5b3c6",
+              "process": "RECO",
+              "recid": "11153",
+              "title": "conffile"
+            },
+            {
+              "cms_confdb_id": "f83bc23f20bdf9090c24e193b7f17112",
+              "process": "HLT",
+              "recid": "11233",
+              "title": "conffile"
```

Comment:
This is not pretty, we'd need to amend `data-curation/cms-YYYY-simulated-datasets/code/dataset_records.py` the following line:

```console
$ git grep conffile
dataset_records.py:236:            configuration_files['title'] = 'conffile'
```

in order to generate "proper" titles.  See e.g. how we did this in 2011 in `data-curation/cms-2011-simulated-datasets/code/create_config_file_records.py`:

```
def get_title(afile):
    "Return suitable title of configuration file."
    return 'Configuration file for ' + get_process(afile) + ' step ' + get_python_filename(afile)
```

We'll do this later with @mantasavas